.github/.jira_sync_config.yaml

```diff
@@ -28,12 +27,12 @@ settings:
   sync_comments: false
   
   # (Optional) (Default: None) Parent Epic key to link the issue to
-  epic_key: "LXD-1251"
+  epic_key: "LXD-2467"
```

Comment:
Will the previously synced stories be moved into the new epic?

Reply:
Unfortunately not, I need to change the parent in Jira. The synchronization however works even if you move the issues in another epic.

Reply:
Ok thanks!